ada/testsuite/ada/nameres.adb

```diff
@@ -52,9 +52,26 @@ procedure Nameres is
       --  Whether to display short images for resolved names
    end record;
 
+   type Reference_Kind is (Any, Subp_Call, Subp_Overriding, Type_Derivation);
+   --  Kind of reference for Find_All_References (FAR) pragmas. Each kind leads
```

Comment:
Funny, I just read an article about Win16 programming, where FAR was an acronym for something else https://en.wikipedia.org/wiki/Far_pointer. I'm not a fan of this acronym, I'd rather you use `Refs_Request` which is clearer and mostly as short. Less terminology